src/commands/BeautifyCommand.ts

```diff
@@ -64,17 +61,16 @@ export class BeautifyCommand extends BaseCommand {
     try {
       return Promise.resolve(JSON.parse(configJson));
     } catch (error) {
-      this.writeError(error.message);
-      this.exitCode = 2;
-      return Promise.reject(error);
+      return this.handleError(error, 2);
     }
   }
 
   private configFile(configFile?: string, filePath?: string) {
     const configExplorer = cosmiconfig("unibeautify", {});
+    const dirPath = path.dirname(filePath || "");
```

Comment:
How about

```javascript
const dirPath = filePath && path.dirname(filePath);
```

And then don't call `.search` if `dirPath` is falsy. We don't want Unibeautify looking outside of the project directory automatically, unless the user explicitly provides a config file path outside of the current working directory.

Reply:
Actually we don't need it like that after all.  I thought it wasn't working because of that, but that's not the case:  [If the value is a directory, that's where the search starts. If it's a file, the search starts in that file's directory](https://github.com/davidtheclark/cosmiconfig#searchfrom).

I added `stopDir: filePath` to the cosmiconfig options, that way it only searches in the one directory.